client/src/layout/Layout.jsx

```diff
@@ -0,0 +1,25 @@
+import { useState } from "react";
+import MobileMenu from "./MobileMenu/MobileMenu";
+import Nav from "./Nav/Nav";
+import './Layout.scss'
+
+const Layout = (props) => {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const toggle = () => {
+    setIsOpen(!isOpen);
+    console.log('clicked');
```

Comment:
now that we know `toggle` works, we can remove this console.log before merging into development

Reply:
👍